tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_controller_component.ts

```diff
@@ -189,4 +191,18 @@ export class LinkedTimeFobControllerComponent {
 
     this.onSelectTimeChanged.emit(newLinkedTime);
   }
+
+  deselectFob(fob: Fob) {
+    if (fob === Fob.END) {
+      this.onSelectTimeChanged.emit({...this.linkedTime, end: null});
+      return;
+    }
+
+    if (this.linkedTime.end !== null) {
+      this.onSelectTimeChanged.emit({start: this.linkedTime.end, end: null});
```

Comment:
Could you transfer some of the helpful documents from the PR description to here? Either as function-level comment or just comments sprinkled throughout the function implementation:

> When in range selection(which means we have a start and an end fob) deselecting a fob will leave the remaining fob in place. This means we switch to single selection. If the end fob is deselected we simply remove it. However, if the start fob is deselected we must change the end fob to the start fob before removing the end fob. This gives the effect that the start fob was remove. Lastly when in single selection deselecting the fob toggles the feature entirely.

Reply:
Added.